src/components/ReportActionItem/IOUPreview.js

```diff
@@ -101,121 +101,124 @@ const defaultProps = {
     isHovered: false,
 };
 
-const IOUPreview = (props) => {
```

Comment:
I converted `IOUPreview` to a class component to call `Report.openIOUPreview` only once in `componentDidMount`. This prevents that command from being called 6 times as we were doing before.